1-introduction/introduction_Task4.py

```diff
@@ -0,0 +1,20 @@
+from sys import maxsize
+
+
+def file_size(size_in_bytes):
+    def converter(value, unit='B'):
```

Comment:
Good solution! Although in my opinion it is redundant here and complicates code readability. I'll leave it up to you, but if you want, make it simpler and use a regular loop.